public/index.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html lang="en-us">
   <head>
-      <title>Course Authoring | edX</title>
+      <title>Course Authoring | <%= process.env.SITE_NAME %></title>
```

Comment:
@arbrandes This uses static configuration (`process.env.SITE_NAME`) instead of [runtime configuration](https://github.com/openedx/frontend-wg/issues/103) (`getConfig().SITE_NAME`). Am I right in presuming that runtime config isn't available in this file?

Reply:
Yes, you are right, just static configuration. Also, this approach use all MFEs.

Reply:
@ihor-romaniuk, right.  However, Kyle's point is relevant: in order to truly fix the problem, we'll have to introduce a way to get the `SITE_NAME` dynamically.

We can use `frontend-app-account` as a template.  See the `<Head />` component that was [introduced there](https://github.com/openedx/frontend-app-account/blob/master/src/index.jsx#L28) for this very purpose.  It makes use of the `<Helmet />` component [to modify the page title](https://github.com/openedx/frontend-app-account/blob/master/src/head/Head.jsx#L11) with the dynamically-obtained site name.

Do you mind doing that here as well?  Otherwise, the change is not very useful to current users.